datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/logos.py

```diff
@@ -0,0 +1,77 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import click
+from PIL import Image
+
+from ...constants import NOT_TILES, get_root
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_waiting
+from ...utils import get_valid_tile_checks
+
+
+REQUIRED_IMAGES = {
+    'avatars-bot.png': (128, 128),
+    'saas_logos-bot.png': (200, 128),
+    'saas_logos-small.png': (120, 60)
+}
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Validate logos files'
+)
+@click.argument('check', required=False)
+def logos(check):
+
+    """Validate logo files."""
+
+    if check:
+        if check in NOT_TILES:
+            echo_success(text='Check {} is a blacklisted integration.'.format(check))
+            return
+        else:
+            checks = [check]
+    else:
+        checks = sorted(get_valid_tile_checks())
+
+    errors = dict()
+    error_checks = set()
+
+    for check in checks:
+        path_to_check_logos = os.path.join(get_root(), check, 'logos')
+
+        echo_info('Validating logos for {}'.format(path_to_check_logos))
```

Comment:
Let's not print something for every check, only for checks with errors, and print a quick recap at the end for number of success, and failures. You can look at the other validate tasks to see how it's done.

Reply:
I don't really want to skip log lines for checks when they are successful because I think its helpful to see whats being validated, and since there is a blacklist I'd like to know if the check I'm expecting to have validation against, is actually running.

Reply:
Well for that you do have the option to validate one check only by specifying it as argument.
The thing is, you rarely look at what's successful anyway, so it only adds noise for other failures that you are looking for.

Reply:
I wound up addressing this and only print errors + if a check is in the blacklist. Let me know what you think.